collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -105,45 +104,8 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-
```

Comment:
@nribeka Yes. `Robolectric` works by creating shadow object of a class. I tried running test without moving [this](https://github.com/opendatakit/collect/pull/553/files#diff-587fa336330bb907832356ee77ed08d1R107) line of the top but test was breaking due to _AssertionError_. After searching a lot I found that that `setContentView()` should be on the top of `onCreate()` method. That's the reason for shifting code.